src/app/api/gql/fragments/fragment-dataset-data-sample.graphql

```diff
@@ -0,0 +1,11 @@
+fragment DatasetDataSample on DataQueryResult {
```

Comment:
Let's call this `DataFrame` - "sample" makes it sound that it's only used for the "Overview" tab, but I think this will be a fragment that we will reuse in many places.